### PR TITLE
Add basic jest code coverage

### LINKED ISSUE #1973: Add basic unit test code coverage reports
**Is your feature request related to a problem? Please describe.**
Currently we have no way to measure our unit test code coverage. Jest has a built-in method to generate a coverage report which is displayed in the CLI and within a generated folder (`/coverage).

**Describe the solution you'd like**
We should leverage the code coverage capabilities of jest.
In the future, we can choose to build on top of this initial code coverage (i.e. implement codecov, coveralls, etc. add code coverage badges and integrate with ci/cd)

### PR DESCRIPTION
**Pull request by ZachTRice**
## Description

Fixes #1973

- [x] Add jest test coverage (+ handle Windows)
- [x] Update developing documentation
- [x] Update complex npm scripts to use run-s

## Checklist

- [x] I have read the [CONTRIBUTING](https://github.com/nasa-gibs/worldview/blob/master/.github/CONTRIBUTING.md) doc
- [x] I have added necessary documentation (if applicable)
- [x] I have added tests that prove my fix is effective or that my feature works (if applicable)
- [x] Any dependent changes have been merged and published in downstream modules (if applicable)

@nasa-gibs/worldview

### CODE REVIEW

File: doc/testing.md [hunk 1]
```diff
@@ -1,19 +1,24 @@
...
+Unit tests are run using the Jest testing framework. Unit tests ran as part of the main testing suite, i.e. `npm test`.
+Use `npm test:unit` to run unit tests individually.
+_Note:_ Make sure to run `npm run getcapabilities` and
 `npm run build:config` first to build the configuration.
 
-To test against a different time zone, run `npm run test:unit:tz`. This
-currently doesn't work in Windows.
+To test against a different time zone, run `npm run test:unit:tz`. _Note:_ This currently doesn't work in Windows.
+
+To view the unit test code coverage, run `npm test:coverage`.
```
Comment (Benjaki2) [inline doc/testing.md]: `npm test:coverage` didn't work for me. Needed to do `npm run test:coverage`, same with `npm test:unit` above.

Comment (ZachTRice) [reply doc/testing.md]: Ah, yeah, I guess you would need to use the `run` command on anything other than vanilla `npm test`

Comment (ZachTRice) [reply doc/testing.md]: Updated, thanks!

**Comment by ZachTRice** [comment]
I think in time we should def add a github badge and continue to build coverage tools on top of this. This is just a starting point to get the ball rolling.

**Pull request merged**